docs/examples/advanced/nef_algorithm.ipynb

```diff
@@ -156,7 +156,7 @@
     "    voltage and v=1 is the firing threshold.\n",
     "    \"\"\"\n",
     "    spikes = []\n",
-    "    for i in range(len(v)):\n",
+    "    for i, _ in enumerate(v):\n",
```

Comment:
Why was this change made? I see at least one other place in the code where it's still `range()`, but not sure why there but not here.

Reply:
This tripped the `consider-using-enumerate` pylint rule. This happens here and not in other places because `i` is being used to index into `v`. I do not actually use the value from enumerate in this case because `v[i]` appears on the left-hand side of an equation, so using that value would have no effect. It also appears on the right hand side of some lines, so I could use it there, but it feels confusing to have the same quantity using two different identifiers in the same local block. Similarly, it's much easier/cleaner to make this change than it is to add a `pylint disable` comment.